st3/sublime_lib/view_stream.py

```diff
@@ -165,18 +179,24 @@ def _seek(self, offset):
 
     @guard_validity
     def seek_start(self):
-        """Move the cursor in the view to before the first character."""
```

Comment:
I like this short style for single-line docstrings more. But this is mostly your code, so it's just a suggestion. PEP-257 also mandates this, however.

By the way, I also prefer to have the first line of the docstring on the same line as the opening punctuation because I have a plugin that folds all docstrings when I open Python files except for the summary line. It's capable of folding the empty first line up to the actual start of the summary line, but when I want to unfold it using fold arrows, ST will unfold the collapsed whitespace first instead of the remaining body of the docstring because that comes first.
And I think having the body in a separate paragraph is more visually pleasing (applies to the previous change in this file).

Reply:
This makes sense.

Previously, I've been wrapping comments after column 80. With semantic newlines, most comment lines won't be this long in the first place, so I think it makes sense to wrap at 100 instead (like non-comment lines). This will prevent most summary lines from wrapping, which should be helpful when reading folded code.